File: 백준/Silver/1476. 날짜 계산/날짜 계산.cc
#include <iostream>

using namespace std;

int main() {
    int e=1,s=1,m=1,cnt = 1;
    int E,S,M;
    cin >> E >> S >> M;
    while(true)
    {
        if(e == E && s == S && m == M) break;
        
        e++;
        s++;
        m++;
        if(e == 16) e = 1;
        if(s == 29) s = 1;
        if(m == 20) m = 1;
        cnt++;
    }
    cout << cnt;
}

File: 백준/Silver/1476. 날짜 계산/날짜 계산.java
import java.util.*;

public class Main {
    public static void main(String args[]) {
      Scanner sc = new Scanner(System.in);
      int e=1,s=1,m=1,cnt=1;
      int E = sc.nextInt(),S = sc.nextInt(),M = sc.nextInt();
      while(true)
      {
          if(e == E && s == S && m == M) break;
        
          e++;
          s++;
          m++;
          if(e == 16) e = 1;
          if(s == 29) s = 1;
          if(m == 20) m = 1;
          cnt++;
      }
      System.out.println(cnt);
    }
}